frontend/src/search/SearchResultEntry.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Represents a option element inside the dropdown of the autocomplete search.
+ */
+export interface SearchResultEntry {
```

Comment:
Add a comment to every (/some) attributes. I don't know what `subLabel` should be.